spec/services/calculators/multiples/separate_proceedings_spec.rb

```diff
@@ -52,4 +52,14 @@
       expect(subject.spent_date).to eq(expiry_date)
     end
   end
+
+  describe '#without_relevant_orders' do
+    let(:check_group) { instance_double(CheckGroup, disclosure_checks: [disclosure_check, relevant_order]) }
```

Comment:
That's a good observation, thinking...

Reply:
@zheileman that is correct, if 1 conviction has only a relevant order, that relevant order is discarded, quoting example:

>Jack age 21 was convicted of possession of an offensive weapon on 25/01/17 and received a **two-year conditional discharge order** (**_relevant order_**). Jack’s conviction would become spent on 25/01/19. 
> On 10/12/18 Jack was convicted of battery and received a 12-month community order. This conviction would become spent on 10/12/20. 
>Although he has been convicted of a further offence while within the rehabilitation period of the first offence, 
the first conviction will not be subject to the ‘drag on effect’ because a conditional discharge is a relevant order and so it will not be taken into account. 
Therefore, in this scenario, the first conviction would become ‘spent’ on 25/01/19; 
And the second conviction would become ‘spent’ on 10/12/20.